eclipse-collections-code-generator/src/main/resources/api/list/primitiveList.stg

```diff
@@ -125,6 +131,29 @@ public interface <name>List extends Reversible<name>Iterable
     {
         throw new UnsupportedOperationException("Default method to prevent breaking backwards compatibility");
     }
+<if(primitive.specializedStream)>
+
+    /**
+     * @since 10.0
+     */
+    Spliterator.Of<name> spliterator();
+
+    /**
+     * @since 10.0
+     */
+    default <name>Stream primitveStream()
+    {
+        return StreamSupport.<type>Stream(this.spliterator(), false);
+    }
+
+    /**
+     * @since 10.0
+     */
+    default <name>Stream primitveParallelStream()
+    {
```

Comment:
I thought these were going to be called stream() and parallelStream()?

Reply:
I made the following research:
spliterator.ofint implements Spliterator<Integer> so there is no problem to use a primitive spliterator as normal boxed spliterator:
https://docs.oracle.com/javase/8/docs/api/java/util/Spliterator.OfInt.html

IntStream is not implementing stream. it is implementing basestream, as also stream. so technically they are brothers:
https://docs.oracle.com/javase/8/docs/api/java/util/stream/IntStream.html
https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html

I don't see how in the same function i can return something which is also a stream and also an intstream.
If we want tu support also boxed streams we will need to save the name to stream and parallelstream. I am open to suggestions

Reply:
but i found a typo in primitive :-)